dataBase/DBCHERRY.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python 
+import sqlite3 as sql 
```

Comment:
we don't need it. Why we need it? please explain.
We already have the DBcreator to create the database.